.github/workflows/ci.yaml

```diff
@@ -1091,6 +1091,7 @@ jobs:
     needs:
       - info
       - pytest
+    timeout-minutes: 10
```

Comment:
I've seen it fail quite a few times recently too.

It normally takes under 10 seconds. You can probably even reduce this to 1, or maybe 2 minutes.

Reply:
not really needed, a small hickup is fine to handle.
It just prevents it running for hours.